app/humans.txt

```diff
@@ -14,3 +14,4 @@ Ali Felskia
 Alireza
```

Comment:
Should we change this whole file to just the listing of Cumulus OX people because none of those other people work on this anymore? And delete the Dev Seed name

Reply:
That would be up to Lauren since she is the Product Owner. She may want to update this. I just added the Cumulus OX team to it because it was missing.

Reply:
And let's update this while we're at it